app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_like_counter
+    post.increment!(:like_counter)
+  end
```

Comment:
#### Optional 
*you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_like_counter` method every time a new post has been saved to database, then make the `update_like_counter` method private

```ruby

  after_save :update_like_counter

  private

  def update_like_counter
    author.increment!(:posts_counter)
  end
```